Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM gcr.io/google.com/cloudsdktool/cloud-sdk:latest
+ENV NODE_VERSION=14.18.0
+RUN  set -ex; \
+  apt-get update -y; \
+  apt-get install -y curl
+RUN curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.38.0/install.sh | bash
+ENV NVM_DIR=/root/.nvm
```

Comment:
I like this approach you've landed on, it might be worth pulling this into a sub-folder, e.g., `containers/`, in case we end up needing a couple?

Reply:
Nah, I don't think we're going to need multiple containers here.  It's really just Cloud SDK with mah nodes.